### DISCUSSION TITLE
Organization Settings Editor

### DISCUSSION THREAD

**Discussion by mikiekelly** [EsriPS/arcgis-assistant-feedback#32] [Ideas]
I would love to see an editable json view of the standard organization tab within ArcGIS Online/Enterprise. This would be useful to facilitate fine grained changes much like with content items.

**Comment by jpeterson** [comment]
Hey @mikiekelly 👋🏼 

This is an interesting idea! Would you mind sharing a few example use cases/workflows you would want to accomplish by editing org/portal settings in Assistant? The surface area of that API is pretty huge, so I want to be sure we understand what you're most interested in.

**Reply by mikiekelly** [reply]
Hi @jpeterson :wave:

Thanks for following up! I have encountered a number of bugs after updates which restrict certain modifications to the self org settings that come from: https://<org>.maps.arcgis.com/sharing/rest/portals/self?culture=en-ca&f=json&token=<token>

That would be the main ask. A couple of other things would be nice too:

Org Resources (so you can easily update the new Home Page app) - this could have a similar interface to item resources: https://<org>.maps.arcgis.com/sharing/rest/portals/self/resources?start=1&num=100&f=json&token=<token>

Members Properties - On occasion I find that updates are needed for specific users especially when it comes to idpUsername: https://<org>.maps.arcgis.com/sharing/rest/community/users/<username>?ts=1635953918352&f=json&token=<token>

Just my wish list! I have used Python to update these on occasion but would be nice to have another option.

Cheers,

Mikie
